rest-api/dao/questionnaire_response_dao.py

```diff
@@ -353,9 +355,21 @@ def from_client_json(self, resource_json, participant_id=None, client_id=None):
     if questionnaire.status == QuestionnaireDefinitionStatus.INVALID:
       raise BadRequest("Submitted questionnaire that is marked as invalid: questionnaire ID %s" %
                        questionnaire.questionnaireId)
+    authored = None
+    if fhir_qr.authored and fhir_qr.authored.date:
+      authored = fhir_qr.authored.date
+
+    language = None
+    if fhir_qr.extension:
+      for ext in fhir_qr.extension:
+        if 'iso21090-ST-language' in ext.url:
+          language = ext.valueCode[:2]
```

Comment:
When I work on other tickets, I accidentally found that there are some values of `primary_language` like below:

`select distinct primary_language from participant_summary`
```
en
es
"en"
"es"
```
I doubt the double quotes are from the xml extension, but I didn't dig more yet, just FYI. I think before we know what happened, maybe we need to take care of this to make sure it will not cause any exception.

Reply:
In this case the FHIR object strips the double quotes and only the language code is stored.  I was dealing this the other day, where another field ended up wrapped with double quotes. I believe the quotes get captured when code is accessing data directly from a JSON string.  I saw this issue in a unittest recently.

Reply:
Nice